bcda/models/models.go

```diff
@@ -265,6 +270,32 @@ func AssignAlphaBeneficiaries(db *gorm.DB, aco ACO, acoSize string) error {
 	return db.Exec(s).Error
 }
 
+type CCLF8File struct {
+	gorm.Model
+	Name            string
```

Comment:
```suggestion
	Name            string `gorm:"not null;unique"`
```
This should always exist and will be unique.  If it isn't we have a duplicate import and should throw an error probably.